packages/did-ethr-resolver/test/read-attributes.test.ts

```diff
@@ -9,23 +9,25 @@ import {
   IUpdateData,
   PubKeyType,
 } from '@ew-did-registry/did-resolver-interface';
+import { Methods } from '@ew-did-registry/did';
 import {
   Operator, signerFromKeys, getProvider, ConnectedSigner,
 } from '../src';
 
 import { deployRegistry } from '../../../tests/init-ganache';
 
+const keys = new Keys({
+  privateKey: '49d484400c2b86a89d54f26424c8cbd66a477a6310d7d4a3ab9cbd89633b902c',
+  publicKey: '023d6e5b341099c21cd4093ebe3228dc80a2785479b8211d20399698f61ee264d0',
+});
+
 describe('[DID-RESOLVER-READ-ATTRIBUTES]', function () {
   this.timeout(0);
-  const keys = new Keys({
-    privateKey: '49d484400c2b86a89d54f26424c8cbd66a477a6310d7d4a3ab9cbd89633b902c',
-    publicKey: '023d6e5b341099c21cd4093ebe3228dc80a2785479b8211d20399698f61ee264d0',
-  });
   let operator: Operator;
   let registry: string;
   const identity = '0x37155f6d56b3be462bbd6b154c5E960D19827167';
   const validity = 10 * 60 * 1000;
-  const did = `did:ethr:${identity}`;
+  const did = `did:${Methods.Erc1056}:${identity}`;
```

Comment:
the formal name of the ERC 1056 based DID registry is `ethr`.

Reply:
We access this constant through enumeration to avoid typos and have possibility to change registry name without changing code in did-lib clients